xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/Videos/WebHome.xml

```diff
@@ -42,50 +42,92 @@
-    $services.localization.render('help.videos.videoCard2.topic2'),
-    $services.localization.render('help.videos.videoCard2.topic3')
+    {
+      'url': 'https://extensions.xwiki.org/xwiki/bin/view/Extension/Flamingo%20Theme%20Application#HHowtoselectatheme',
+      'label': "$services.localization.render('help.videos.videoCard2.topic1')"
+    },
+    {
+      'url': 'https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/GettingStarted/ChangingTheLogoAndThePanels#HChangingyourwiki27spanels',
+      'label': "$services.localization.render('help.videos.videoCard2.topic2')"
+    },
+    {
+      'url': 'https://extensions.xwiki.org/xwiki/bin/view/Extension/Dashboard%20Macro#HAddinggadgets',
+      'label': "$services.localization.render('help.videos.videoCard2.topic3')"
+    }
   ],
   'url': 'https://www.youtube.com/embed/zX1Itoh3E68',
   'duration': '5 min'
 }, {
   'title': $services.localization.render('help.videos.videoCard3.title'),
   'topics': [
-    $services.localization.render('help.videos.videoCard3.topic1'),
-    $services.localization.render('help.videos.videoCard3.topic2'),
-    $services.localization.render('help.videos.videoCard3.topic3')
+    {
+      'url': 'https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/GettingStarted/CreatingAPage',
+      'label': "$services.localization.render('help.videos.videoCard3.topic1')"
+    },
+    {
+      'url': 'https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/GettingStarted/EditingAPage',
+      'label': "$services.localization.render('help.videos.videoCard3.topic2')"
+    },
+    {
+      'url': 'https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/Features/KeyboardShortcuts#HViewMode',
+      'label': "$services.localization.render('help.videos.videoCard3.topic3')"
+    }
   ],
   'url': 'https://www.youtube.com/embed/onenEu21FIk',
   'duration': '6 min'
 }, {
   'title': $services.localization.render('help.videos.videoCard4.title'),
   'topics': [
-    $services.localization.render('help.videos.videoCard4.topic1'),
-    $services.localization.render('help.videos.videoCard4.topic2')
+    {
+      'url': 'https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/Features/Imports#HImportingOfficedocuments',
+      'label': "$services.localization.render('help.videos.videoCard4.topic1')"
+    },
+    {
+      'url': 'https://extensions.xwiki.org/xwiki/bin/view/Extension/TOC+Macro#HExample1:Simple',
+      'label': "$services.localization.render('help.videos.videoCard4.topic2')"
+    }
   ],
   'url': 'https://www.youtube.com/embed/csQVSRMVclM',
   'duration': '4 min'
 }, {
   'title': $services.localization.render('help.videos.videoCard5.title'),
   'topics': [
-    $services.localization.render('help.videos.videoCard5.topic1'),
-    $services.localization.render('help.videos.videoCard5.topic2')
+    {
+      'url': $xwiki.getURL('Help.Applications.WebHome'),
+      'label': "$services.localization.render('help.videos.videoCard5.topic1')"
```

Comment:
Why is the `$xwiki.getURL` not in quotes but `$services.localization.render` is?